docs/sources/writing-guide/shortcodes/index.md

```diff
@@ -143,6 +143,54 @@ This shortcode inserts a lists of links to the page's subpages and includes the
 {{</* section withDescriptions="true"*/>}}
 ```
 
+## `docs/reference` shortcode
+
+The `docs/reference` shortcode offers more flexible linking than the Hugo builtin `relref` shortcode.
+When content from one repository is published to more than one documentation set, the `docs/reference` shortcode lets you specify appropriate links for each doc set.
+
+For example, alerting documentation is sourced from the Grafana repository and published to both Grafana documentation and Grafana Cloud documentation.
+Any link from alerting documentation to dashboard documentation, for example, which is also published to both Grafana and Grafana Cloud, needs to be specific as to which destination is appropriate for the product documentation.
+Grafana Cloud alerting documentation should link to Grafana Cloud dashboard documentation, and Grafana alerting documentation should link to Grafana dashboard documentation.
+
+The content of the `docs/reference` shortcode must be within opening and closing tags.
+
+| Parameter             | Description                                                                                                                                                                                                                                                               | Required |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `label`               | The label for the shortcode. This is the text that you enter in the link instead of an http link or file path.                                                                                                                                                            | yes      |
+| `project path prefix` | Designates the destination project. For example, for Grafana, use `/docs/grafana/`, for Mimir, use `/docs/mimir/`.                                                                                                                                                        | yes      |
+| `reference`           | The path to the file. It can include `<SOMETHING VERSION>`, which is either taken from front matter of (which file) or falls back to being inferred from the version of the page. This enables the use of absolute paths that resolve correctly, irrespective of version. | yes      |
```

Comment:
I still think it is important to reference the relref parameters as described in Hugo documentation. To me, the reference in the example below does not look like "the path to the file" because it has no file extension. It looks more like a path to a page, but that is also not correct and will cause confusion if people adopt that mental model. It is strictly the same as the parameter described in https://gohugo.io/content-management/cross-references/#use-of-ref-and-relref.